functions/wav_time.h

```diff
@@ -0,0 +1,143 @@
+#ifndef FUNCTIONS_WAV_TIME_H
+#define FUNCTIONS_WAV_TIME_H
+
+#include "wav_effect.h"
+#include "int.h"
+#include "../blades/blade_base.h"
+
+// Usage: WavLen<EFFECT_NAME>  
+// EFFECT_NAMENUM : any off the available effect form the style edittor
+// Returns length of wav file returned as const int in [ms]
+// returned value: INTEGER
+//
+template<BladeEffectType EFFECT>
+class WavLen {
+ public:
+  void run(BladeBase* blade) {
+    found_ = false;
+    eff_name_ = wav_effect.GetEffectName(EFFECT);//find correspondig SoundEffectName for BladeEffectType
+    for (size_t i = 0; i < NELEM(wav_players); i++) {
```

Comment:
I'm not a big fan of iterating over the wav players to find stuff.
What if there are two playing the same effect?

Reply:
Is that even possible (clash maybe??). What would the current effect be. Surely one of those is an older effect still playing?